examples/quickstarts/helidon-quickstart-se/src/main/java/io/helidon/examples/quickstart/se/GreetService.java

```diff
@@ -122,7 +123,26 @@ private void updateGreetingFromJson(JsonObject jo, ServerResponse response) {
      */
     private void updateGreetingHandler(ServerRequest request,
                                        ServerResponse response) {
-        request.content().as(JsonObject.class).thenAccept(jo -> updateGreetingFromJson(jo, response));
+        request.content().as(JsonObject.class)
+            .thenAccept(jo -> updateGreetingFromJson(jo, response))
+            .exceptionally((Throwable ex) -> {
+                if (ex.getCause() instanceof JsonException){
+                    String error = String.format("Error with JSON: %s", ex.getCause().toString());
+                    JsonObject jsonErrorObject = JSON.createObjectBuilder()
+                        .add("error", error)
+                        .build();
+                    response.status(Http.Status.BAD_REQUEST_400).send(jsonErrorObject);
+                } else {
+                    JsonObject jsonErrorObject = JSON.createObjectBuilder()
+                        .add("error", ex.getLocalizedMessage())
+                        .build();
+                    request.next(ex);
+                }
+
+                ex.printStackTrace();
```

Comment:
Exception.printStackTrace() should be avoided, instead this should be done with a logger. There is no logger in this class, but it's a good opportunity to add one: `private static final Logger LOGGER = Logger.getLogger(GreetService.class.getName())`

Also, I think it makes sense the create a separate method for the exception handling to make it re-usable. Even though here it's only used once, it shows a good practice as part of the quickstart.

E.g.

```java
    private static <T> T processErrors(Throwable ex, ServerRequest request, ServerResponse response) {
        if (ex.getCause() instanceof JsonException) {
            String error = String.format("Error with JSON: %s", ex.getCause().toString());
            JsonObject jsonErrorObject = JSON.createObjectBuilder()
                    .add("error", error)
                    .build();
            response.status(Http.Status.BAD_REQUEST_400).send(jsonErrorObject);
        } else {
            JsonObject jsonErrorObject = JSON.createObjectBuilder()
                    .add("error", ex.getLocalizedMessage())
                    .build();
            response.status(Http.Status.INTERNAL_SERVER_ERROR_500).send(jsonErrorObject);
            request.next(ex);
        }
        LOGGER.log(Level.FINE, "Exception occurred while processing payload", ex);
        return null;
    }
```

```java
    private void updateGreetingHandler(ServerRequest request,
                                       ServerResponse response) {
        request.content().as(JsonObject.class)
                .thenAccept(jo -> updateGreetingFromJson(jo, response))
                .exceptionally(ex -> processErrors(ex, request, response));
    }
```

Reply:
My latest commit addresses this change. Is sending the error message sufficient or should I add a more customized message? Thanks @romain-grecourt !

Reply:
I'm not sure about sending the exception message back in the response body, I'll let @tomas-langer share his opinion about this.

I have a few more comments about the current changes, see my next review.